runnable/generate_request_outputs.py

```diff
@@ -286,10 +22,20 @@ def generate_request_outputs(write_legacy:bool, write_validator_checkpoint:bool)
                 write_legacy = True
                 write_validator_checkpoint = True
 
+            current_time_ms = TimeUtil.now_in_millis()
             if write_legacy or write_validator_checkpoint:
```

Comment:
this is a good opportunity to remove the flags write_legacy and write_validator_checkpoint. Just update everything at once please